src/app/containers/Timestamp/relativeTimestamp.js

```diff
@@ -0,0 +1,31 @@
+const relativeTime = receivedTimestamp => {
+  const currentTime = Date.now();
+  const timeDifferenceMilliseconds = currentTime - receivedTimestamp;
+  const toMinuteDivisor = 1000 * 60;
+  const timeDifferenceMinutes = parseInt(
+    timeDifferenceMilliseconds / toMinuteDivisor,
+    10,
+  );
+
+  if (timeDifferenceMinutes > 10 * 60 || timeDifferenceMilliseconds < 0) {
+    return null;
+  }
+
+  let timeDiff;
+  let unit;
+
+  if (timeDifferenceMinutes >= 60) {
+    timeDiff = timeDifferenceMinutes / 60;
+    unit = timeDiff > 1 ? 'hours' : 'hour';
+  } else if (timeDifferenceMinutes <= 1) {
+    timeDiff = 1;
+    unit = 'minute';
+  } else {
+    timeDiff = timeDifferenceMinutes;
+    unit = 'minutes';
+  }
```

Comment:
I think you could avoid the middle `else if` block and just have an `else` that looks similar to your opening `if`.

```js
} else {
  timeDiff = (timeDifferenceMinutes <= 1) ? 1 : timeDifferenceMinutes;
  unit = (timeDiff === 1) ? 'minute' : 'minutes'; 
}
```

Reply:
thanks 👍

Reply:
tbf it felt more readable before... 🙈

Reply:
Ha 😄I think we're just halfway through a refactor.

My next suggestion would be to have this on line 16:

```js
return constructTimestamp(timeDifferenceMinutes);
```

...and move lines 16-27 into that new function.